src/endpoints/metrics.js

```diff
@@ -58,10 +81,10 @@ metricsRouter.get("/createdCount/:resourceType", (req, res, next) => {
     { $unwind: "$resourceMetadata" },
     {
       $match: {
-        "resourceMetadata.versions.0.timestamp": {
-          $gt: new Date(req.query.startDate),
-          $lt: new Date(req.query.endDate),
-        },
+        "resourceMetadata.versions.0.timestamp": getDateQuery(
```

Comment:
Not exactly relevant to this PR, but can you say more about the `resourceMetadata.versions.0` part of the query?

Reply:
yes, will add a comment